docs/vars.md

```diff
@@ -0,0 +1,96 @@
+Configurable Parameters in Kargo
+================================
+
+#### Generic Ansible variables
+
+You can view facts gathered by Ansible automatically
+[here](http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts).
+
+Some variables of note include:
+
+* *ansible_user*: user to connect to via SSH
+* *ansible_default_ipv4.address*: IP address Ansible automatically chooses.
+  Generated based on the output from the command ``ip -4 route get 8.8.8.8``
+
+#### Common vars that are used in Kargo
+
+* *calico_version* - Specify version of Calico to use
+* *calico_cni_version* - Specify version of Calico CNI plugin to use
+* *docker_version* - Specify version of Docker to used (should be quoted
+  string)
+* *etcd_version* - Specify version of ETCD to use
+* *ipip* - Enables Calico ipip encapsulation by default
+* *hyperkube_image_repo* - Specify the Docker repository where Hyperkube
+  resides
+* *hyperkube_image_tag* - Specify the Docker tag where Hyperkube resides
+* *kube_network_plugin* - Changes k8s plugin to Calico
```

Comment:
The description looks incorrect. There is a correct one given below, please don't repeat here.